docs/libcurl/opts/CURLOPT_RESOLVE.3

```diff
@@ -37,10 +37,12 @@ list of \fBstruct curl_slist\fP structs properly filled in. Use
 to clean up an entire list.
 
 Each single name resolve string should be written using the format
-HOST:PORT:ADDRESS where HOST is the name libcurl will try to resolve, PORT is
-the port number of the service where libcurl wants to connect to the HOST and
-ADDRESS is the numerical IP address. If libcurl is built to support IPv6,
-ADDRESS can of course be either IPv4 or IPv6 style addressing.
+HOST:PORT:ADDRESS... where HOST is the name libcurl will try to
```

Comment:
i think this would be easier read as HOST:PORT:ADDRESS[,ADDRESS]...

Reply:
Agree. Fixed now.